contracts/RelayHub.sol

```diff
@@ -35,7 +35,7 @@ contract RelayHub is IRelayHub {
     mapping(address => StakeInfo) public stakes;
 
     //maps relay manager address to the relay data
-    mapping(address => RelayData) public relayData;
+    mapping(address => RelayManagerData) public relayData;
```

Comment:
the name `RelayInfo` was preferable, but was already taken (`rif-relay-client/src/types/RelayInfo.ts`)